### PR TITLE
feat: update team member by (superadmin)

### PR DESCRIPTION
**Pull request by chukssomzzy**
## Description

Implemented API To `PATCH` team members (superadmin)

​

## Related Issue (Link to issue ticket)
["FEAT: Update Team Member By ID (super Admin)"](https://github.com/hngprojects/hng_boilerplate_python_fastapi_web/issues/732)

​

## Motivation and Context

To allow update of team member information within a company. 

​

## How Has This Been Tested?

- [ ] Written Unit test with pytest 
- [ ] Extensively tested on postman

​

### Update team member by id (superadmin)
image

### Tested related pr (add team member) 
image

​

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
      ​

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: api/v1/routes/team.py [hunk 1]
```diff
@@ -68,13 +92,13 @@ async def add_team_members(
     """
     if member.name.strip() == "" or member.role.strip() == "" or member.description.strip() == "" or member.picture_url.strip() == "":
```
Comment (zxenonx) [inline api/v1/routes/team.py:93]: What's the objective here?

**Review by zxenonx — COMMENTED** [review]
@chukssomzzy Thanks. I left a comment.

File: api/v1/routes/team.py (hunk 1 above)
Comment (chukssomzzy) [reply api/v1/routes/team.py:93]: That not my changes, I don't know.

**Review by zxenonx — APPROVED** [review]
Thanks @chukssomzzy

**Review by CodewithSegNet — APPROVED** [review]

**Pull request merged**